src/pe_source/cybersixgill.py

```diff
@@ -0,0 +1,301 @@
+"""Collect Cybersixgill data."""
+
+# Standard Python Libraries
+from datetime import date, datetime, timedelta
+import logging
+import sys
+
+from .data.pe_db.db_query import (
+    get_breaches,
+    get_data_source_uid,
+    get_orgs,
+    insert_sixgill_alerts,
+    insert_sixgill_breaches,
+    insert_sixgill_credentials,
+    insert_sixgill_mentions,
+    insert_sixgill_topCVEs,
+)
+from .data.sixgill.api import get_sixgill_organizations
+from .data.sixgill.source import (
+    alerts,
+    alias_organization,
+    creds,
+    cve_summary,
+    mentions,
+    root_domains,
+    top_cves,
+)
+
+# Set todays date formatted YYYY-MM-DD and the start_date 30 days prior
+TODAY = date.today()
+DAYS_BACK = timedelta(days=30)
+START_DATE = str(TODAY - DAYS_BACK)
+END_DATE = str(TODAY)
+DATE_SPAN = f"[{START_DATE} TO {END_DATE}]"
+
+# Set dates to YYYY-MM-DD H:M:S format
+NOW = datetime.now()
+BACK = timedelta(days=30)
+FROM_DATE = (NOW - BACK).strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Is there any reason why you can't reuse `DAYS_BACK`?
```suggestion
FROM_DATE = (NOW - DAYS_BACK).strftime("%Y-%m-%d %H:%M:%S")
```

Reply:
Committed this suggestion here: https://github.com/cisagov/pe-reports/pull/117/commits/90dbfcfdbc974049b2b1dabc4bb69a79b96c757c